salt/metalk8s/kubeadm/init/etcd/local.sls

```diff
@@ -7,6 +7,24 @@
 {% if ip_candidates %}
 {% set host = ip_candidates[0] %}
 
+{% set endpoint  = host_name ~ '=https://' ~ host ~ ':2380' %}
+
+{%- set ca_cert = salt['mine.get']('*', 'kubernetes_etcd_ca_server').values()[0] %}
```

Comment:
`kubernetes_etcd_ca_server` is a bit of an odd name for what is actually the CA certificate itself (if I got this correctly). Not in scope of this PR to fix it, I guess, but nonetheless, something to reconsider.

Reply:
Tracked by #890